feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -22,7 +22,8 @@ export interface ProductExtended extends Product {
 }
 
 export interface ConfiguratorAttributeProductCardComponentOptions {
-  preventAction?: boolean;
+  preventAllActions?: boolean;
+  preventRemoveAction?: boolean;
```

Comment:
Could you comment those attributes, and also consider renaming them?
I find this very confusing, not easy to understand this component, because the naming preventAllActions, preventRemoveAction suggests that the latter is a specialisation of the first.
This is not true: 
preventAllActions rather means that the controls are disabled.
preventRemoveAction means that a button is not rendered at all.

So preventAllActions can be true while  preventRemoveAction is false

Reply:
ok